# Temperature Sensor Message
std_msgs/Header header
string sensor_id
float32 temperature
float32 humidity
float32 pressure
bool is_operational